sdl2promela/stop_condition/parser.py

```diff
@@ -220,7 +226,7 @@ def _parse_filter_out_statement(
     return FilterOutStatement(_parse_expression(statement.children[0]))
 
 
-def parse_stop_condition(text: str) -> StopConditionModel:
+def parse_stop_condition(text: str) -> Optional[StopConditionModel]:
```

Comment:
What is the practical impact of this? We are returning Union of Null and StopConditionModel.. instead of StopConditionModel, which could be Null... right? Why?

Reply:
`None`, if the return type is just type, the None is reporter as an error by `mypy`